doc/nrf/ug_matter_gs_ecosystem_compatibility_testing.rst

```diff
@@ -0,0 +1,56 @@
+.. _ug_matter_gs_ecosystem_compatibility_testing:
+
+Testing Matter ecosystem compatibility
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO:
+
+    * add info about SPECIFIC hubs and how to connect our DKs with them; we can link to the blog post and/or add links to the producer page
+    * add link to the ecosystems page in the getting started
+
+    * add information about mobile controllers to Testing page, as per comment in https://github.com/nrfconnect/sdk-nrf/pull/10189/files#
+
+Nordic Semiconductor's Matter platform and Matter applications integrated in the |NCS| are follow the most recent versions of the Matter protocol and are interoperable with several external Matter ecosystems.
+This interoperability relies on Matter's :ref:`multiple fabrics feature <ug_matter_overview_multi_fabrics>` and leverages Matter's support for using different wireless protocols for different use cases.
+
+You can test the interoperability of the Matter solution in the |NCS| with a selection of compatible Matter ecosystems by following the `Testing the nRF Connect platform with Apple, Google and Samsung ecosystems`_ tutorial on DevZone.
+This will allow you to set up and test the following scenario with Apple, Google, and Samsung ecosystems.
+
+.. figure:: images/matter_ecosystem_compatibility.png
+   :alt: Example of Matter ecosystem compatibility
+
+   Example of Matter ecosystem compatibility
+
+Prerequisites
+*************
+
+You need the following pieces of hardware to set up and test interoperability with Apple, Google, and Samsung ecosystems:
+
+* Matter over Thread development kit: 1x nRF52840 DK or 1x nRF5340 DK
+* Matter over Wi-Fi development kit: 1x nRF7002 DK
+* For Apple Home ecosystem:
+
+  * 1x Apple iPhone (iOS 16.1 or higher)
+  * 1x Apple HomePod Mini (iOS 16.1 or higher)
+
+* For Google Home ecosystem:
+
+  * 1x Android smartphone with Google Home app installed (it can be the same that is used for the SmartThings app)
+  * 1x Google Nest Hub 2nd generation or 1x Google Nest Hub Max (the tutorial uses Google Nest Hub 2nd generation)
+
+* For SmartThings ecosystem:
+
+  * 1x Android smartphone with SmartThings app installed (it can be the same that is used for the Google Home app)
+  * 1x Smart Home Hub working with SmartThings and supporting Wi-Fi and Thread technologies (for example, Aeotec Smart Home Hub, as used in the tutorial)
```

Comment:
Do we need to list ecosystem's devices? They will change over time, and already Google and Apple support multiple devices. Personally i would remove this section.

I like Kamil's suggestion:
"The bare minimum set up to create Matter network and test Matter accessory is one Matter over Thread or Matter over Wi-Fi nRF development kit, and one set of devices creating the ecosystem."